Assignments/stu5/data-modeling.sql

```diff
@@ -35,3 +35,36 @@ END;
 /**
  * Ex. 12
  */
+-- This script was generated by a beta version of the ERD tool in pgAdmin 4.
+-- Please log an issue at https://redmine.postgresql.org/projects/pgadmin4/issues/new if you find any bugs, including reproduction steps.
+BEGIN;
+
+
+CREATE TABLE IF NOT EXISTS public.bank
+(
+    bank_id serial NOT NULL,
+    bank_account text,
```

Comment:
To tighten up this model, we can change the `bank_account text` column to `bank_account_id integer` and create a another new table called `bank_account` to store all the bank accounts, and link this table to the `bank` table.   Make sense?